crates/egui-wgpu/src/winit.rs

```diff
@@ -538,20 +539,21 @@ impl Painter {
                             b: clear_color[2] as f64,
                             a: clear_color[3] as f64,
                         }),
-                        store: true,
+                        store: wgpu::StoreOp::Store,
                     },
                 })],
                 depth_stencil_attachment: self.depth_texture_view.as_ref().map(|view| {
                     wgpu::RenderPassDepthStencilAttachment {
                         view,
                         depth_ops: Some(wgpu::Operations {
                             load: wgpu::LoadOp::Clear(1.0),
-                            store: true,
+                            store: wgpu::StoreOp::Discard,
```

Comment:
In all other places, `true` got replaced with `Store`, and `false` with `Discard`. Is this different intentionally?

Reply:
well observed and yes it is! I should leave an explaining comment.
Typically the depth buffer used during rendering the main egui screen isn't needed afterwards. There might be a usecase for it but I have a pretty hard time thinking of one. In absence of that this unnecessarily costs bandwidth

Reply:
So it being `true` until now has been unnecessary then, right?

Reply:
yep, I think so!

Reply:
Cool, thanks!